ansible/roles/stack-sunbird/templates/ml-core-service.env

```diff
@@ -0,0 +1,77 @@
+# ML core Service Config
+
+# Port on which the service will listen to
+APPLICATION_PORT={{ml_core_port|default("3000")}} 
+
+#Application ENV example local/dev/production
+APPLICATION_ENV={{ml_core_node_env|default("development")}}
+
+# Setting to turn on/off console logs
+ENABLE_CONSOLE_LOGGING={{ml_core_enable_debug_logging | default("ON")}}
```

Comment:
please disable debug log off by default, if in need, we'll override in private

Reply:
done